### ISSUE TITLE
PascalVOC-AUG pseudo labels

### ISSUE THREAD

**Issue by lkdci** [tensorflow/tpu#986]
Hi thanks for your work.
Is the PascalVOC-AUG pseudo labels used in the paper, "Rethinking Pre-training and Self-training", publicity available?
If not, are there any instructions on how to create those? Couldn't find some details in the paper about which model was used to predict the labels? Does the pseudo label is defined by argmax of the predictions or by a fixed threshold?
